lib/versions-ext.xml

```diff
@@ -1515,6 +1515,15 @@
 					<license-url>https://oss.oracle.com/licenses/CDDL-1.1</license-url>
 				</licenses>
 			</library>
+			<library>
+				<file-name>lib/portal/txw2.jar</file-name>
+				<version>2.3.1</version>
+				<project-name>TXW2 Runtime</project-name>
+				<licenses>
+					<license>CDDL+GPL 1.1</license>
+					<license-url>http://glassfish.java.net/public/CDDL+GPL_1_1.html</license-url>
+				</licenses>
+			</library>
 			<library>
```

Comment:
@marianoalvarosaiz, It seems lack of `<project-url>` label. I noticed only our libs("Liferay Jenkins Results Parser" and "Liferay Poshi Core") doesn't exist the label, other third party libs own the label. Please add this. You may add "https://eclipse-ee4j.github.io/jaxb-ri/" as `<project-url>`. Refer to https://mvnrepository.com/artifact/org.glassfish.jaxb/txw2/2.3.5

Reply:
@marianoalvarosaiz,

> the problem is that the project home page was included in version 2.3.5, but version 2.3.1, the one we are using, doesn't include a home page, see https://mvnrepository.com/artifact/org.glassfish.jaxb/txw2/2.3.1. Even the license between both versions is different.

Yes, I noticed this difference. But I didn't noticed the License difference. Thanks for pointing out it. 

I think the third-party libs all should own project-url when it was published. We'd better to find it and add it.
 
By checking the below infos form txw2-2.3.1-sources.jar\META-INF\MANIFEST.MF.
```
Build-Version: JAXB RI 2.3.1
Git-Revision: ad5fa4c697632694cbcfa80177707db908cd98b2
Git-Url: scm:git:git://java.net/jaxb~v2/jaxb-txw-parent/txw2
```
I find https://github.com/javaee/jaxb-v2. The source should be https://github.com/javaee/jaxb-v2/tree/2.3.1/jaxb-ri/txw/runtime/src/main/java/com/sun/xml/txw2. 

Based on https://github.com/javaee/jaxb-v2, I think we may use **https://javaee.github.io/jaxb-v2** as `<project-url>`. It will use the same <project-url> with https://github.com/liferay-core-infra/liferay-portal/blob/db05c0ab8815811becb4d6b4267640c4eb577c66/lib/versions-ext.xml#L1189-L1200. And jaxb-runtime 2.3.1 also didn't include home page(https://mvnrepository.com/artifact/org.glassfish.jaxb/jaxb-runtime/2.3.1)

By the way, I thought we should use 2.3.8 at the beginning. But when seeing your [comment](https://liferay.atlassian.net/browse/LPS-144149?focusedCommentId=2466313), I checked its MANIFEST.MF from jaxb-runtime.jar. It required 2.3.1. Before this, I compared 2.3.1 with 2.3.8. Their codes are the same, except copyright and `@deprecated` annotation on some methods. I just compared 2.3.1 with 2.3.5. The result is the same as 2.3.8 compared result. This is wired.